app/views/doorkeeper/applications/index.html.haml

```diff
@@ -0,0 +1,26 @@
+= render "layouts/manage/page_title", title: t(:title, scope: 'pages.manage.doorkeeper'), docs: 'https://coderit.org/hackathon-manager/docs/busses' do
+  = link_to t(:new, scope: 'doorkeeper.applications.index'), new_oauth_application_path, class: "btn btn-sm btn-outline-secondary"
+
+.mb-4
+  %table.table.table-striped.table-hover
+    %thead
+      %tr
+        %th
+          = t(:name, scope: 'doorkeeper.applications.index')
+        %th
+          = t(:callback_url, scope: 'doorkeeper.applications.index')
+        %th
+          = t(:confidential, scope: 'doorkeeper.applications.index')
+
+
+    %tbody
+      - @applications.each do |application|
+        %tr
+          %td
+            %strong
+            = link_to(application.name, oauth_application_path(application))
```

Comment:
Suggestion: bold name link to match link styling in other areas of HM (questionnaires, schools, etc.)
Note that it could be a bit more standardized: some have icons (magnifying glass) while others have no icon.